<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: century gothic;
font-size: 24px;
}

QWidget
{
background-color: qlineargradient(spread:pad, x1:0.221, y1:0.772909, x2:1, y2:0, stop:0 rgba(70, 169, 255, 255), stop:1 rgba(255, 255, 255, 255))
}

#NavBar
{
background-color: #333;
}

QLabel
{
color:white;
background: transparent;
font-family: Lemon Milk;
font-style: italic;
}

#AccueilBouton
{
background: transparent;
color: white;
font-style: normal;
font-size: 20px;
font-family: Lemon Milk;
}

#Logo
{
background: transparent;
}

#ConsultationRapportBouton
{
background: transparent;
color: skyblue;
font-style: normal;
font-size: 20px;
font-family: Lemon Milk;
}

#selectionPraticien{
background: white;
color: black;
border-radius: 10px;
}

#dateRapport{
background: white;
color: black;
border-radius: 10px;
}

#motif{
background: white;
color: black;
border-radius: 10px;
}

#bilan{
background: white;
color: black;
border-radius: 10px;
}

#selectionMedicamentNom{
background: white;
color: black;
border-radius: 10px;
}

#selectionMedicamentQuantite{
background: white;
color: black;
border-radius: 10px;
}

#boutonAjouterMedicament{
background: white;
color: black;
border-radius: 10px;
}

#tableauOffreMedicaments{
background: white;
color: black;
border-radius: 10px;
}

#NavBar
{
background-color: #333;
}

#PraticienLabel
{
color:black;
font-family: Lemon Milk;
font-style: normal;
}

#DateRapportLabel
{
color:black;
font-family: Lemon Milk;
font-style: normal;
}

#BilanLabel
{
color:black;
font-family: Lemon Milk;
font-style: normal;
}

#ListeLabel
{
color:black;
font-family: Lemon Milk;
font-style: normal;
}

#MotifLabel
{
color:black;
font-family: Lemon Milk;
font-style: normal;
}

#IdRapportLabel
{
color:black;
font-family: Lemon Milk;
font-style: normal;
}

#validerIdRapport
{
color:black;
background: white;
font-family: Lemon Milk;
font-style: normal;
}

#RetourConsultation
{
background: white;
color:black;
font-family: Lemon Milk;
font-style: normal;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QFrame" name="NavBar">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="Logo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qrc/image/logogsb.png</normaloff>qrc/image/logogsb.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="LogoNom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>APPLIVISITEUR</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>0</y>
         <width>141</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="AccueilBouton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Accueil</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>241</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPushButton" name="ConsultationRapportBouton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Consultation de rapport</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>561</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="PraticienLabel">
        <property name="text">
         <string>Praticien :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="BilanLabel">
        <property name="text">
         <string>Bilan :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DateRapportLabel">
        <property name="text">
         <string>Date du rapport :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="motif">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPlainTextEdit" name="bilan">
        <property name="inputMethodHints">
         <set>Qt::ImhMultiLine</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="MotifLabel">
        <property name="text">
         <string>Motif :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTableWidget" name="tableauOffreMedicaments"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ListeLabel">
        <property name="text">
         <string>Liste d'Ã©chantillons :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="selectionPraticien">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dateRapport">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>222</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="validerIdRapport">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="IdRapportLabel">
      <property name="text">
       <string>Id du rapport :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="retourConsultation">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>790</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logoqrc.qrc"/>
 </resources>
 <connections/>
</ui>
